src/os/ui/featureedit.js

```diff
@@ -978,6 +994,12 @@ os.ui.FeatureEditCtrl.prototype.loadFromFeature = function(feature) {
       this['centerIcon'] = icon;
     }
 
+    if (config.fill && config.fill.color) {
+      this['fillColor'] = os.color.toHexString(config.fill.color);
+      var opacity = os.color.toRgbArray(config.fill.color);
+      this['fillOpacity'] = opacity[3];
+    }
+
```

Comment:
I think changing this to a string reference will fix the issue I saw with border color not being in sync with the fill color.

```
if (config['fill'] && config['fill']['color']) {
  this['fillColor'] = os.color.toHexString(config['fill']['color']);
  var opacity = os.color.toRgbArray(config['fill']['color']);
  this['fillOpacity'] = opacity[3];
}
```

Reply:
This fix has been applied and partially fixed the problem. The preview on the screen now updates but the fill color within the Feature Edit form does not update.